chroma-manager.conf.template

```diff
@@ -192,6 +192,17 @@ server {
         gzip_types application/json;
     }
 
+    location /api/task {
+        proxy_set_header Host $http_host;
```

Comment:
```suggestion
    auth_request /auth;
```

Do we want auth on this endpoint? Seems like not everyone should be able to read / submit tasks.